src/web/IntegrationTests/Stubs/StubFavouritesBasketStore.cs

```diff
@@ -7,6 +7,11 @@ namespace DfE.EmployerFavourites.Web.IntegrationTests.Stubs
 {
     public class StubFavouritesBasketStore : IApprenticeshipFavouritesBasketStore
     {
+        public Task RemoveAsync(Guid basketId)
+        {
+            throw new NotImplementedException();
```

Comment:
This will need an implementation and should be tested as part of the Application/Commands/SaveApprenticeshipFavouriteBasketCommandHandler.cs tests